src/Nethermind/Nethermind.Evm/VirtualMachine.cs

```diff
@@ -2119,6 +2119,10 @@ void UpdateMemoryCost(ref UInt256 position, in UInt256 length)
                         {
                             _state.UpdateStorageRoot(contractAddress, Keccak.EmptyTreeHash);
                         }
+                        else if (_state.IsDeadAccount(contractAddress))
```

Comment:
Are we sure this won't create consensus issues on different transactions and blocks?